docs/src/examples/components/Dropdown/Types/DropdownExampleMultiple.shorthand.tsx

```diff
@@ -28,4 +29,25 @@ const getA11ySelectionMessage = {
   onRemove: item => `${item} has been removed.`,
 }
 
+const getA11yStatusMessage = ({
```

Comment:
is this the right thing to do?

I'd suggest encapsulate this logic as an accessibility behavior /  internal implementation for `Dropdown` that will just receive the relevant strings / methods as props:

```typescript
const getStatusMessageFromCount = (resultCount: number) =>
  `${resultCount} result${ resultCount === 1 ? ' is' : 's are'} available, use up and down
arrow keys to navigate. Press Enter key to select. Press left and right arrow keys to
navigate through the selected options.` 

getA11yStatusMessage={{
  default: getStatusMessageFromCount,
  noResult: 'No results are available.',
}}

Reply:
same comment to the other instances

Reply:
I don't think we want to do that. It's a message we don't want to save in our code due to localisation considerations.